pkg/channel/consolidated/reconciler/controller/controller.go

```diff
@@ -89,11 +89,8 @@ func NewController(
 		logger.Panicf("unable to process Kafka channel's required environment variables: %v", err)
 	}
 
-	if env.Image == "" {
```

Comment:
Is there a reason this was removed? is it because `envconfig.Process` will return an error and you're addressing it already so this won't happen?

Reply:
> is it because envconfig.Process will return an error and you're addressing it already so this won't happen?

yes